packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -141,3 +146,38 @@ const product = new servicecatalog.CloudFormationProduct(this, 'MyFirstProduct',
   ]
 });
 ```
+
+### Adding a product to a portfolio
+
+You add products to a portfolio to manage your resources at scale.  After adding a product to a portfolio,
+it creates a portfolio-product association, and will become visible from the portfolio side in both the console and service catalog CLI.
+A product can be added to multiple portfolios depending on your resource and organizational needs.
+
+```ts fixture=portfolio-product
+portfolio.addProduct(product);
+```
+
+## Constraints
+
+Constraints define governance mechanisms that allow you to manage permissions, notifications, and options related to actions end users can perform on products,
+Constraints are applied on a portfolio-product association.
+Using the cdk, if you do not explicitly associate a product to a portfolio and add a constraint, it will automatically add an association for you.
+
+There are rules around plurariliites of constraints for a portfolio and product. For example, you can only have one `TagUpdate`
+constraint applied to a portfolio-product association. 
+If a misconfigured constraint is added, `synth` will fail with an error message.
+
+Read more at [Service Catalog Constraints](https://docs.aws.amazon.com/servicecatalog/latest/adminguide/constraints.html).
+
+### Tag update constraint
+
+Tag update constraints allow or disallow end users to update tags on resources associated with an AWS Service Catalog product upon provisioning. By default, tag updating is not configured.
+If tag updating is allowed, then new tags associated with the product or portfolio will be applied to provisioned resources during a provisioned product update.
+
+```ts fixture=portfolio-product
+portfolio.addProduct(product);
+
+portfolio.allowTagUpdates(product, {
+  allowUpdatingProvisionedProductTags: true,
+});
```

Comment:
Let's show an example without this property passed first:

```suggestion
portfolio.allowTagUpdates(product);
```

And then below:

```ts
// to disable tag updates:
portfolio.allowTagUpdates(product, {
  allowUpdatingProvisionedProductTags: false,
});
```